app/src/main/AndroidManifest.xml

```diff
@@ -9,14 +9,17 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
-        <activity android:name=".SignUpActivity"></activity>
-        <activity android:name=".SignInActivity">
+        <activity android:name=".SelectionActivity">
```

Comment:
The main view must be the main activity, and the main activity will have the responsibility to check if the user is connected or not, if the  user is not connected the main activity will switch to the sign in activity

And from signing activity the user will have the possibility to switch to sign up activity

Reply:
it was the doctor app concept.. but now it's photo app... but on it

Reply:
my idea of a main activity is different tho.. thought it's an activity that u should land on after everything like signin or signiup like that.. 
but cool